db/views/hmis_client_projects_v01.sql

```diff
@@ -0,0 +1,29 @@
+(
+  -- client related to project through enrollment
+  SELECT
+    "Client"."id" AS client_id,
+    "Project"."id" AS project_id,
+    "Enrollment"."id" AS enrollment_id
+  FROM
+    "Client"
+    INNER JOIN "Enrollment" ON "Enrollment"."DateDeleted" IS NULL
+    AND "Enrollment"."data_source_id" = "Client"."data_source_id"
+    AND "Enrollment"."PersonalID" = "Client"."PersonalID"
+    INNER JOIN "Project" ON "Project"."DateDeleted" IS NULL
+    AND "Project"."data_source_id" = "Enrollment"."data_source_id"
+    AND "Project"."ProjectID" = "Enrollment"."ProjectID"
+  WHERE
+    "Client"."DateDeleted" IS NULL
+)
```

Comment:
I think we should probably limit this view to only HMIS clients (as opposed to all clients which includes Warehouse clients). It will make it much smaller. @eanders what do you think? Or is this something you want available to the warehouse?

If limiting to the HMIS, we would join data source and only include clients where DataSource.hmis is non-null.

Reply:
So, I've been debating if we'd want to use this in the warehouse since seeing it.  I'm still on the fence.  I think probably not.  
I'd be ok limiting to just the HMIS Data Source

Reply:
Ok. Theron and I also discussed adding columns here for data_source_id and EnrollmentID, so that enrollment-related records like Service, CustomAssessment, etc can have a relation to `project` using a composite key to this, through EnrollmentID. (Currently their relation to project drops WIPs, so we almost always override it in a way that is not performant)

Reply:
Yep, fair enough. I think it would be about half or a third of the enrollments in this data source, but I'd have to look at prod to see.